linera-views/src/common.rs

```diff
@@ -174,6 +198,249 @@ pub trait KeyValueStoreClient {
     }
 }
 
+/// A key-value store that splits the big values into multiple keys.
```

Comment:
I would suggest to move this code to a separate file (say `big_value.rs`) with its own unit tests to cover all the desired behaviors.